python/GafferUI/VectorDataWidget.py

```diff
@@ -418,32 +423,41 @@ def editSignal( self ) :
 
 		return self.__editSignal
 
+	## A signal emitted when the user right clicks selected cells.
+	# Slots should accept ( VectorDataWidget, IECore.MenuDefinition ) arguments,
+	# modify the given menu and return it.
```

Comment:
The equivalent functionality elsewhere (e.g. `PlugValueWidget.popupMenuSignal()`) doesn't require that the menu be returned. I think we're better off sticking to existing convention so folks don't get confused.

Reply:
Fixed in 8211b7e5d43506ebc2f28a7af9764466c9039746, and comment fixed and squashed into 71a0621e4ce0494c03be56e1315fa5a62a9ff1f2